Gemfile

```diff
@@ -21,6 +21,8 @@ gem 'jbuilder', '~> 2.0'
 # bundle exec rake doc:rails generates the API under doc/api.
 gem 'sdoc', '~> 0.4.0', group: :doc
 gem 'pg'
+gem 'carrierwave'
```

Comment:
@leonardk1 Before I did this feature I did my research on most of the plugins out there and I found this to be more promising than paperclip